.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,14 +17,23 @@ https://{{head.ref}}--60f9b557105290003b387cd5.chromatic.com
 ## Description
 Closes <ticket>
 
-## Testing done
-
+## QA Checklist
+- [ ] Component maitains 1:1 parity with design mocks
+- [ ] Text is consistent with what's been provided in the mocks
+- [ ] Component behaves as expected across breakpoints
+- [ ] Accessibility expert has signed off on code changes (if applicable. If not applicable provide reason why)
+- [ ] Designer has signed off on changes (if applicable. If not applicable provide reason why)
+- [ ] Tab order and focus state work as expected
+- [ ] Changes have been tested against screen readers (if applicable. If not applicable provide reason why)
+- [ ] New components are covered by e2e tests / updates to existing components are covered by existing test suite
```

Comment:
^e2e tests; updates (recommend semi-colon for greater clarity; the '/' can be read as "e2e tests OR updates")

Reply:
This one is a personal nit-pick, do with it what you want. ;)

Reply:
nah you're right, OR is better here.